lib/Dialect/FIRRTL/Transforms/BlackboxMemory.cpp

```diff
@@ -97,7 +97,7 @@ static InstanceOp createInstance(OpBuilder builder, Location loc,
 
   return builder.create<InstanceOp>(loc, resultTypes, moduleName,
                                     builder.getArrayAttr(resultNames),
-                                    instanceName);
+                                    instanceName, builder.getArrayAttr({}));
```

Comment:
Not sure if using the default argument for annotations would help here, then we can skip the empty,  ``builder.getArrayAttr({})``. Many other similar use case for default empty annotations.

Reply:
Good thought. I'll add a default builder here that can be used to avoid this change. Most of the other situations of this should already be fixed.

Reply:
New builder with defaults added in 4babfdc8 and I switched this to use it in cf2359db. That makes that transform forward compatible with this PR.